wallet/wallet-rpc-lib/src/rpc/mod.rs

```diff
@@ -221,6 +222,33 @@ impl<N: NodeInterface + Clone + Send + Sync + 'static> WalletRpc<N> {
             .await?
     }
 
+    pub async fn find_timestamps_for_staking(
```

Comment:
> This function also needs documentation

Functions in this `WalletRpc` struct don't seem to have much documentation. And since this function just calls `find_timestamps_for_staking` in wallet-controller, I decided to put the documentation there instead, "for consistency". Again, this is a copy-paste of the documentation for `WalletCommand::FindTimestampsForStaking`, so it's triplicated now.